rllib/policy/view_requirement.py

```diff
@@ -0,0 +1,58 @@
+from gym.spaces import Box, Space
```

Comment:
A simple (data) class to specify view requirements (from underlying data) for forward passes, postprocessing sample batches and train batches.